### ISSUE TITLE
improve website links to the source install guide

### ISSUE THREAD

**Issue by gubuntu** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#382]
The install guide link on http://qgis.org/en/site/forusers/download.html points to master, which has changed radically from the 2.x releases, so it could be made clearer on the download page by provided links to the install buides for the different releases.

http://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTALL.html

**Comment by DelazJ** [comment]
@gubuntu:

> The install guide link on http://qgis.org/en/site/forusers/download.html points to master, 

Can you provide more precise information on the link/text we should look at, plz? Can't find a particular _install guide_ nor mention of master on the download page

**Comment by gubuntu** [comment]
I usually build various versions of QGIS and I had set up a new machine so installed all the xenial dependencies from that link. I then spent many hours manually figuring out all the other dependencies I needed to build 2.8, 2.14 and 2.16 before I realised the doc was specifically for master, which is all Qt5 and Python3. So I tried the doc on a release branch (e.g. http://htmlpreview.github.io/?https://raw.githubusercontent.com/qgis/QGIS/release-2_16/doc/INSTALL.html) and voilà, there was the information I needed. 

So, maybe have a list of docs to link to or a dropdown that generates the link for the version you want to build.
